Gemfile

```diff
@@ -3,7 +3,7 @@ source "https://rubygems.org"
 gem "gds-sso"
 gem "govuk_app_config"
 gem "plek"
-gem "rails", "~> 5.2"
+gem "rails", "~> 6.0.3"
```

Comment:
The default Rails Gemfile has this line:

``` ruby
gem "rails", "~> 6.0.3", ">= 6.0.3.2"
```

to ensure that security release is applied so we should follow that

Reply:
:sparkles: